### DISCUSSION TITLE
Unable to install NodeJs for windows

### DISCUSSION THREAD

**Discussion by ShivaliLN** [nodejs/help#4704] [Q&A] [answered]
Hi Team,

I am unable to install windows 64-bit installer -- It says Page not found. Please help.

**Comment by ShivaliLN** [comment, marked as answer]
installed from here https://nodejs.org/dist/latest-v18.x/

**Comment by vkruoso** [comment]
This still an issue for all non-latest version, which most automated tools use.

**Reply by askingtherightquestion** [reply]
I just stumbled upon the same problem. Were you able to find out anything in the meantime?

**Reply by askingtherightquestion** [reply]
I've created a separate discussion under https://github.com/nodejs/node/discussions/47191 for anyone wondering.
